driver-core/src/main/java/com/datastax/driver/core/ProtocolOptions.java

```diff
@@ -231,4 +251,11 @@ public AuthProvider getAuthProvider() {
         return authProvider;
     }
 
+    /**
+     * Whether or not connecting to server with protocol version which is currently in beta is allowed.
+     * @return {@code true} if connecting with beta protocol version is allowed, {@code false} otherwise.
+     */
+    public boolean isAllowBetaProtocolVersions() {
```

Comment:
Good point. I'm pretty sure @adutra mentioned it in a different context`allowsBetaProtocolVersions`, sorry I missed it.